src/include/daos_srv/evtree.h

```diff
@@ -65,6 +64,11 @@ struct evt_desc {
 	uint32_t			dc_magic;
 	/** The DTX entry in SCM. */
 	umem_id_t			dc_dtx;
```

Comment:
I just noticed that this is poorly packed as well.  umem_id_t is 64-bit aligned.   If you put your new field above dc_magic, it will be a net-zero addition